libs/subliminal_patch/subtitle.py

```diff
@@ -61,8 +61,10 @@ class Subtitle(Subtitle_):
     pack_data = None
     _guessed_encoding = None
     _is_valid = False
+    use_original_format = False
+    format = "srt" # default format is srt
 
-    def __init__(self, language, hearing_impaired=False, page_link=None, encoding=None, mods=None):
+    def __init__(self, language, hearing_impaired=False, page_link=None, encoding=None, mods=None, original_format=False):
```

Comment:
Really nice parameter but you don't set it to True anywhere... ;-)

Reply:
Yeah I am working on front end and kinda of learning how react work.. this pr is still WIP

Reply:
Roger I'll wait for your final version. Thanks!